aws/image/files/etc/systemd/system/agent-protocol-forwarder.service

```diff
@@ -4,6 +4,9 @@ After=network.target
 
 [Service]
 ExecStart=/usr/local/bin/agent-protocol-forwarder -kata-agent-namespace /run/netns/podns -kata-agent-socket /run/kata-containers/agent.sock
+Restart=on-failure
+RestartSec=5s
+
```

Comment:
The new issue created by you https://github.com/confidential-containers/cloud-api-adaptor/issues/388 provides the reasoning behind the failure.
Let's remove this change from this PR and tackle it as part of #388